### PR TITLE
Nano : expose onnx trace setting to user

### PR DESCRIPTION
**Pull request by rnwang04**
In some cases, user may want to change the  opset_version to enable some complex operator in pytorch, we may expose the setting to users.
Also in quantize, we should add these parameters(https://github.com/intel-analytics/BigDL/issues/4893).

### CODE REVIEW

File: python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py [hunk 1]
```diff
@@ -367,6 +368,8 @@ def trace(model: nn.Module,
                             backend. 'openvino' and 'onnxruntime' are supported for now.
         :param onnxruntime_session_options: The session option for onnxruntime, only valid when
                                             accelerator='onnxruntime', otherwise will be ignored.
+        :param opset_version: opset_version for export_to_onnx, only valid when
+                                            accelerator='onnxruntime', otherwise will be ignored.
```
Comment (TheaperDeng) [inline python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py:372]: I think `accelerator="openvino"` will also be affected by this opset_version? no? @zhentaocc

Comment (zhentaocc) [reply python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py]: I suppose highest opset version should work.

File: python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py [hunk 2]
```diff
@@ -354,7 +354,8 @@ def quantize(model,  # remove the type requirement for type checking
     def trace(model: nn.Module,
               input_sample=None,
               accelerator=None,
-              onnxruntime_session_options=None):
+              onnxruntime_session_options=None,
+              opset_version=11):
```
[the remark this answers is not in the document]
Comment (zhentaocc) [reply python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py]: why not use latest opset as  default?

File: python/nano/src/bigdl/nano/deps/onnxruntime/onnxruntime_api.py [hunk 3]
```diff
@@ -15,7 +15,8 @@
 #
 
 
-def PytorchONNXRuntimeModel(model, input_sample=None, onnxruntime_session_options=None):
+def PytorchONNXRuntimeModel(model, input_sample=None,
+                            onnxruntime_session_options=None, **kwargs):
     """
         Create a ONNX Runtime model from pytorch.
```
Comment (TheaperDeng) [inline python/nano/src/bigdl/nano/deps/onnxruntime/onnxruntime_api.py:22]: add some comment to tell future developers what may be contained in kwargs?

Comment (TheaperDeng) [reply python/nano/src/bigdl/nano/deps/onnxruntime/onnxruntime_api.py:22]: same things to some of other changes as well

Comment (rnwang04) [reply python/nano/src/bigdl/nano/deps/onnxruntime/onnxruntime_api.py:23]: fix

**Comment by rnwang04** [comment]
http://10.112.231.51:18888/job/BigDL-NanoTests-PR-Validation/274/
http://10.112.231.51:18888/job/BigDL-PRVN-chronos-Python-Spark-2.4-py37-ray-part1/1167/

**Pull request merged**